flink-runtime/src/test/java/org/apache/flink/runtime/jobmaster/JobManagerRunnerTest.java

```diff
@@ -208,13 +200,19 @@ public void testShutDown() throws Exception {
 
 	@Test
 	public void testLibraryCacheManagerRegistration() throws Exception {
-		final JobManagerRunner jobManagerRunner = createJobManagerRunner();
+		final BlobLibraryCacheManager libraryCacheManager = new BlobLibraryCacheManager(
+			VoidPermanentBlobService.INSTANCE,
+			FlinkUserCodeClassLoaders.ResolveOrder.CHILD_FIRST,
+			new String[]{});
+		final JobManagerSharedServices jobManagerSharedServices = new TestingJobManagerSharedServicesBuilder()
+			.setLibraryCacheManager(
+				libraryCacheManager)
```

Comment:
can be on the same line as `setLibraryCacheManager`

Reply:
True, will change it